board/power_saving.h

```diff
@@ -25,22 +25,23 @@ void set_power_save_state(int state) {
       enable = true;
     }
 
-    // Switch CAN transcievers
-    current_board->enable_can_transcievers(enable);
+    // Switch CAN transcievers except for BUS 1, which must remain on to detect CAN based ignition
+    current_board->enable_can_transciever(2U, enable);
+    current_board->enable_can_transciever(3U, enable);
```

Comment:
This doesn't include the 4th CAN transceiver on black panda / uno. Maybe we want to do this in a more generic way? Not entirely sure how yet though.

Reply:
`enable_can_transcievers` can maybe have a second input that is an uint8_t array of buses to leave untouched?